### ISSUE TITLE
NVDA with microsoft edge

### ISSUE THREAD

**Issue by ghost** [nvaccess/nvda#8039]
When typing in the comment body on github with microsoft edge, NVDA keeps saying spelling error, even though I don't make mistakes in my typing. I did the exact same thing in google chrome, as well as firefox and internet explorer, and this problem doesn't occur, whatsoever.

**Comment by PratikP1** [comment]
I am unable to replicate this behavior. Could you please supply the log NVDA log to show this? Thank you.

**Comment by leonardder** [comment]
[**@BEdenhofer**](https://github.com/BEdenhofer) commented on [25 feb. 2018 17:37 CET](https://github.com/nvaccess/nvda/issues/8039#issue-300040222 "2018-02-25T16:37:29Z - Replied by Github Reply Comments"):
> When typing in the comment body on github with microsoft edge, NVDA keeps saying spelling error, even though I don't make mistakes in my typing. I did the exact same thing in google chrome, as well as firefox and internet explorer, and this problem doesn't occur, whatsoever.

**Comment by leonardder** [comment]
@BEdenhofer: Could it be that the language of the spell checker is different from the language you're typing? In that case, the spelling error information can actually be expected.

**Comment by ehollig** [comment]
As we are unable to contact the original author, and @PratikP1 was not able to replicate in https://github.com/nvaccess/nvda/issues/8039#issuecomment-368383152, I am going to close this issue.
